src/js/providers/html5.js

```diff
@@ -483,6 +483,8 @@ define([
                 _this.supportsPlaybackRate = false;
             }
 
+            _setAttribute('preload', 'none');
```

Comment:
That depends on when the **`src`** is set relative to when preload it. This is also one of the reasons why preload is **an `item` or `item.source` property** - so that **video `src` and `preload` attributes** can be set at the same time or correct order, but now the player needs be in control of preloading. So **`item.source.preload`** is completely irrelevant (why are we still setting  **`item.source.preload`**?).